docs/source/getting_started/woodwork_types.ipynb

```diff
@@ -0,0 +1,385 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b95b28c1",
+   "metadata": {},
+   "source": [
+    "# Woodwork Typing in Featuretools\n",
+    "\n",
+    "Featuretools relies on having consistent typing across the creation of EntitySets, Primitives, Features, and feature matrices. Previously, Featuretools used its own type system that contained objects called Variables. Now and moving forward, Featuretools will use an external data typing package for its typing: [Woodwork](https://woodwork.alteryx.com/en/stable/index.html).\n",
+    "\n",
+    "Understanding the Woodwork types that exist and how Featuretools uses Woodwork's type system will allow users to build EntitySets that best represent their data, understand the possible input and return types for Featuretools' Primitives, and understand what features will get generated from a given set of data and primitives.\n",
+    "\n",
+    "Read the [Understanding Woodwork Logical Types and Semantic Tags](https://woodwork.alteryx.com/en/stable/guides/logical_types_and_semantic_tags.html) guide for an in-depth walkthrough of the available Woodwork types that are outlined below.\n",
+    "\n",
+    "For users that are familiar with the old `Variable` objects, the [Transitioning from Variables to Woodwork](link) guide will be useful for converting Variable types to Woodwork types.\n",
```

Comment:
This guide does not exist yet and should be updated to include the link in #1257

Reply:
thank you!